extension/proofgraph.h

```diff
@@ -12,31 +12,36 @@
 #include "godot_cpp/templates/hash_map.hpp"
 #include "godot_cpp/templates/hash_set.hpp"
 #include "godot_cpp/variant/string.hpp"
-
-
+#include "godot_cpp/classes/mesh_instance3d.hpp"
+#include "godot_cpp/classes/mesh.hpp"
+#include "godot_cpp/classes/box_mesh.hpp"
+#include "godot_cpp/classes/text_mesh.hpp"
+#include "godot_cpp/classes/standard_material3d.hpp"
+#include "godot_cpp/variant/color.hpp"
+#include "godot_cpp/classes/cylinder_mesh.hpp"
+#include "godot_cpp/classes/box_shape3d.hpp"
+#include "godot_cpp/classes/collision_shape3d.hpp"
+#include "godot_cpp/classes/static_body3d.hpp"
```

Comment:
Why are we including all of these in the H file if we're not using any of them in the H file? Should be imported in the cpp file where they are actually used.